Misc/NEWS.d/next/Core and Builtins/2022-09-18-08-47-40.gh-issue-96821.Co2iOq.rst

```diff
@@ -0,0 +1 @@
+Fix undefined behaviour in `_testcapimodule.c`.
```

Comment:
```suggestion
Fix undefined behaviour in ``_testcapimodule.c``.
```

I assume you mean to use a literal here, not the default role (italics).

Reply:
Thanks!